DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/ChannelData.h

```diff
@@ -0,0 +1,46 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ChannelData.h
+/// \brief Class to describe fired and  stored channels for the BC and to refer to channel data
+/// \author Alla.Maevskaya@cern.ch
+
+#ifndef _FT0_CHANNELDATA_H_
+#define _FT0_CHANNELDATA_H_
+
+#include <Rtypes.h>
+
+namespace o2
+{
+namespace ft0
+{
+struct ChannelData {
+
+  uint8_t ChId = -1;     //channel Id
```

Comment:
@AllaMaevskaya by initializing the uint8_t wih -1 you are assigning to 256. Is this what you want?

Reply:
no, I fixed it now

Reply:
But you wrote before that the chanID goes from 0 to 208, with ``in8_t`` you cannot cover this range. If you want to initialize with the value which is outside of the allowed range, use `uin8_t` and init with `0xff`